tycho-its/src/test/java/org/eclipse/tycho/test/surefire/JUnit5Test.java

```diff
@@ -77,4 +77,15 @@ public void testJUnit56Runner() throws Exception {
 		assertNumberOfSuccessfulTests(projectBasedir, "bundle.test.JUnit56Test", 1);
 	}
 
+  @Test
+	public void testJUnit59Runner() throws Exception {
+		Verifier verifier = getVerifier("/surefire.junit59/bundle.test", false);
+		verifier.addCliOption("-Drepo-2020-03=" + P2Repositories.ECLIPSE_LATEST.toString());
```

Comment:
This will have to be https://download.eclipse.org/eclipse/updates/4.25-I-builds/ for the moment.

Reply:
yes, I tried it run yesterday locally and today morning again but same result:

```
[INFO] Adding repository https:/download.eclipse.org/eclipse/updates/4.25-I-builds
[ERROR] Internal error: java.lang.RuntimeException: Failed to load p2 repository with ID 'eclipse202003' from location https:/download.eclipse.org/eclipse/updates/4.25-I-builds/: Unable to read repository at https:/download.eclipse.org/eclipse/updates/4.25-I-builds. Connection refused: connect -> [Help 1]
org.apache.maven.InternalErrorException: Internal error: java.lang.RuntimeException: Failed to load p2 repository with ID 'eclipse202003' from location https:/download.eclipse.org/eclipse/updates/4.25-I-builds/
    at org.apache.maven.DefaultMaven.execute (DefaultMaven.java:120)
    at org.apache.maven.cli.MavenCli.execute (MavenCli.java:960)
    at org.apache.maven.cli.MavenCli.doMain (MavenCli.java:293)
    at org.apache.maven.cli.MavenCli.main (MavenCli.java:196)
    at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0 (Native Method)
    at jdk.internal.reflect.NativeMethodAccessorImpl.invoke (NativeMethodAccessorImpl.java:62)
    at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke (DelegatingMethodAccessorImpl.java:43)
    at java.lang.reflect.Method.invoke (Method.java:566)
    at org.codehaus.plexus.classworlds.launcher.Launcher.launchEnhanced (Launcher.java:282)
    at org.codehaus.plexus.classworlds.launcher.Launcher.launch (Launcher.java:225)
    at org.codehaus.plexus.classworlds.launcher.Launcher.mainWithExitCode (Launcher.java:406)
    at org.codehaus.plexus.classworlds.launcher.Launcher.main (Launcher.java:347)
Caused by: java.lang.RuntimeException: Failed to load p2 repository with ID 'eclipse202003' from location https:/download.eclipse.org/eclipse/updates/4.25-I-builds/
```
When use this url in browser I see classic "You're close" web page with info that here is repository. HTTPS vs HTTP problem? or some other

Reply:
`/` is the escape character, so the URL in a .java file must double them: `https:////download.eclipse.org/eclipse/updates/4.25-I-builds`

Reply:
??? escape char is backslash not slash e.g.  \n \t not /n /t. so you escape normal "DOS" path e.g. from c:\windows to c:\\windows or c:/windows.

Reply:
But understood here is in log one slash. in source code are two slashes, So I will try it duplicate it. It is strange for me:(

Reply:
I'm not really sure how, but somewhere in the chain where the URL is passed to the Maven command, the `/` seems te be used as escape, or replaced, or something else... Anyway, it looks like 4 slashes are necessary here.
If by the way you can determine why this is the case and maybe fix it, that would be welcome (in a separate PR).

Reply:
Finally green test locally
```
[INFO] --- maven-surefire-plugin:3.0.0-M5:test (default-test) @ tycho-its ---
[INFO]
[INFO] -------------------------------------------------------
[INFO]  T E S T S
[INFO] -------------------------------------------------------
[INFO] Running org.eclipse.tycho.test.surefire.JUnit5Test
[INFO] Tests run: 5, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 105.685 s - in org.eclipse.tycho.test.surefire.JUnit5Test
[INFO]
[INFO] Results:
[INFO]
[INFO] Tests run: 5, Failures: 0, Errors: 0, Skipped: 0
[INFO]
[INFO]
[INFO] --- maven-jar-plugin:3.2.2:jar (default-jar) @ tycho-its ---
[WARNING] JAR will be empty - no content was marked for inclusion!
[INFO] Building jar: d:\ide\eclipse\tycho-master\git\org.eclipse.tycho\tycho-its\target\tycho-its-3.0.0-SNAPSHOT.jar
[INFO]
[INFO] --- maven-source-plugin:3.2.1:jar-no-fork (default) @ tycho-its ---
[INFO] No sources in project. Archive not created.
[INFO] ------------------------------------------------------------------------
[INFO] BUILD SUCCESS
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  02:12 min
[INFO] Finished at: 2022-08-23T10:15:19+02:00
[INFO] ------------------------------------------------------------------------
```
pushed to PR